apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-save-click.js

```diff
@@ -0,0 +1,16 @@
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_save_click`.
+ *
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export default () => ( {
+	id: 'wpcom-block-editor-save-click',
+	// The first selector is for site editing. The second is for post editing.
+	selector: '.editor-entities-saved-states__save-button, .editor-post-publish-button:not(.has-changes-dot), .editor-post-save-draft', ,
```

Comment:
```suggestion
	selector: '.editor-entities-saved-states__save-button, .editor-post-publish-button:not(.has-changes-dot), .editor-post-save-draft',
```

Whoops, it looks like i accidently added an extra comma in that last suggestion.